addons/sale_project/views/project_task_views.xml

```diff
@@ -124,8 +124,14 @@
                     <field name="project_partner_id" invisible="1"/>
                     <field name="sale_line_id" groups="!sales_team.group_sale_salesman" placeholder="Non-billable" options="{'no_open': True}" readonly="1"/>
                     <field name="sale_line_id" groups="sales_team.group_sale_salesman" options="{'no_create': True}" placeholder="Non-billable" readonly="0"/>
-                    <field name="quantity_percentage" groups="!sales_team.group_sale_salesman" widget="percentage" readonly="1"/>
-                    <field name="quantity_percentage" class="w-25" groups="sales_team.group_sale_salesman" widget="percentage" readonly="0"/>
+                    <label for="quantity_percentage"/>
+                    <div class="col-6">
+                        <field name="quantity_percentage" groups="!sales_team.group_sale_salesman" widget="percentage" readonly="1" class="mw-25"/>
+                        <field name="quantity_percentage" class="mw-25" groups="sales_team.group_sale_salesman"
+                            widget="percentage" decoration-danger="quantity_percentage  &gt; 1 or quantity_percentage &lt; 0" readonly="0"/>
```

Comment:
```suggestion
                        <field name="quantity_percentage" class="mw-25" groups="sales_team.group_sale_salesman"
                            widget="percentage" decoration-danger="quantity_percentage &lt; 0 or 1 &lt; quantity_percentage" readonly="0"/>
```
Little detail, but I think it is more readable when it is ranked in order of magnitude :shrug: 
Also, isn't it possible to use `<` i/o `&lt;`? :thinking: